src/main/scala/mesosphere/marathon/DebugConf.scala

```diff
@@ -82,7 +82,10 @@ class DebugModule(conf: DebugConf) extends AbstractModule {
 
   override def configure(): Unit = {
     //set trace log level
-    conf.logLevel.get.foreach(level => Logger.getRootLogger.setLevel(Level.toLevel(level.toUpperCase)))
+    conf.logLevel.get.foreach { level =>
```

Comment:
The description in `logging_level` says fatal. Please remove that rom the description. Do yo think we should translate fatal to error?